### PR TITLE
21 epidemic

### PR DESCRIPTION
**Pull request by demmerichs**
My first working code on this project :) Hope its not too bad, let me know what you think/can be improved.

### CODE REVIEW

File: modules/GT_StatesCard.php [hunk 1]
```diff
@@ -69,6 +69,35 @@ function stDrawCard($game)
...
   }
 
+  function stEpidemic($game)
+  {
+    $players = GT_DBPlayer::getPlayersInFlight($game, '', $order = 'DESC');
+
+    foreach ($players as $plId => $player) {
+      $plyrBoard = $game->newPlayerBoard($plId);
+      $plyrContent = $game->newPlayerContent($plId);
+      $crewContent = $plyrContent->getContent("crew");
```
Comment (pickardj79) [inline modules/GT_StatesCard.php:79]: This is not part of this ticket, but you could start implementing it now - use constants for content types, like Pavel did for cards. If you set up the framework (e.g. defined all the constants) in this PR then we could just start using them as we continue to build the code.

File: modules/GT_TestGameState.php [hunk 2]
```diff
@@ -227,11 +222,17 @@ function basicShipTiles($color, $variant)
...
+      array_push($tiles, self::newTile(19*6, 8, 6, 0)); // purple alien life support
+      array_push($tiles, self::newTile(40, 6, 8, 180)); // cabin south west
+      array_push($tiles, self::newTile(45, 6, 7, 90)); // cabin west
       $cargo = self::newTile(58, 7, 6, 0); // hazard cargo north
       array_push($tiles, $cargo);
       array_push($tiles, self::newTile(88, 7, 5, 0)); // cannon above that
       array_push($tiles, self::newTile(3, 6, 6, 270)); // battery to west of cargo
       array_push($content, self::newContent($cargo, 'goods', 'red', 1, 1));
+    } elseif ($variant == 3) {
+      array_push($tiles, self::newTile(36, 7, 6, 0));
```
Comment (pickardj79) [inline modules/GT_TestGameState.php:235]: Could you add a comment as to what this tile is?

**Review by pickardj79 — APPROVED** [review]
Looks great - very elegant. A couple of small suggestions - approved and I'll leave it up to you whether or not to make the changes.

Did you test with two cabins with crew that are adjacent but not connected? What about a cabin with crew connected to an empty cabin? I think should both work as intended based on my read of the code but testing would be a good idea too.

**Comment by demmerichs** [comment]
I did test neighboring cabins that were not connected and it worked. But I missed the case of an empty cabin neighboring non-empty, I will add it and test it, thanks.

File: modules/GT_StatesCard.php (hunk 1 above)
Comment (demmerichs) [reply modules/GT_StatesCard.php:79]: I think I will merge this first, and create a new PR, but yes moving the content types to constants makes a lot of sense.

File: modules/GT_TestGameState.php (hunk 2 above)
Comment (demmerichs) [reply modules/GT_TestGameState.php:235]: oh yes, sry forgot that

**Comment by demmerichs** [comment]
closes #21

**Pull request merged**